sdk/python/packages/flet-core/src/flet_core/types.py

```diff
@@ -290,3 +297,20 @@ class MouseCursor(Enum):
     WAIT = "wait"
     ZOOM_IN = "zoomIn"
     ZOOM_OUT = "zoomOut"
+
+
+# Events
+ControlEventFunction = Callable[[ControlEvent], None]
+
+# Wrapper
+Wrapper = Callable[..., Any]
+
+# Args / Kwargs
+Args = List[Any]
+Kwargs = Dict[str, Any]
+
+
+# Protocols
+class SupportsStr(Protocol):
+    def __str__(self) -> str:
+        ...
```

Comment:
What is this for?

Reply:
This is protocol for type hinting. SupportsStr is an object that supports str()